src/mono/browser/test-main.js

```diff
@@ -272,6 +273,14 @@ function configureRuntime(dotnet, runArgs) {
             })
         }
     }
+
+    /*
+    dotnet.withEnvironmentVariable("MONO_LOG_LEVEL", "debug")
+    dotnet.withEnvironmentVariable("MONO_LOG_MASK", "gc")
+    dotnet.withEnvironmentVariable("MONO_GC_DEBUG", "9")
+    dotnet.withEnvironmentVariable("DOTNET_DebugWriteToStdErr", "1")
+    */
```

Comment:
```suggestion
    // dotnet.withEnvironmentVariable("MONO_LOG_LEVEL", "debug")
    // dotnet.withEnvironmentVariable("MONO_LOG_MASK", "gc")
    // dotnet.withEnvironmentVariable("MONO_GC_DEBUG", "9")
    // dotnet.withEnvironmentVariable("DOTNET_DebugWriteToStdErr", "1")
```

Reply:
what makes it better ?

Reply:
1. You simply uncoment only lines you want to
2. If search for the code and get only a preview of few lines, you clearly see they are commented out